static/js/containers/VideoDetailPage_test.js

```diff
@@ -111,56 +104,23 @@ describe('VideoDetailPage', () => {
     let wrapper = await renderPage({editable: true});
     assert.isFalse(wrapper.find(".edit").props().disabled);
     wrapper.find(".edit").props().onClick();
-    assert.isTrue(store.getState().videoDetailUi.editDialog.visible);
+    assert.isTrue(store.getState().commonUi.dialogVisibility[DIALOGS.EDIT_VIDEO]);
   });
 
-  it('edit button has no effect if video is not editable', async () => {
```

Comment:
This test can be modified to work by bringing back `updateVideoStub` and changing the code to:
```python
  it('edits the title and description and submits the data to trigger an update', async () => {
    let wrapper = await renderPage({editable: true});

    const newTitle = 'new title';
    const newDescription = "omg this is the BEST VIDEO";
    const expectedVideo = {
      ...video,
      title: newTitle,
    };
    updateVideoStub.returns(Promise.resolve(expectedVideo));
    await listenForActions([
      actions.videos.patch.requestType,
      actions.videos.patch.successType,
      HIDE_DIALOG,
      INIT_EDIT_VIDEO_FORM,
      INIT_EDIT_VIDEO_FORM,
      SET_EDIT_VIDEO_DESC,
      SET_EDIT_VIDEO_TITLE,
      SHOW_DIALOG,
    ], () => {
      wrapper.find(".edit").props().onClick();
      wrapper.find("#video-title").props().onChange({target: {value: newTitle}});
      wrapper.find("#video-description").props().onChange({target: {value: newDescription}});
      wrapper.find('Dialog').at(1).props().onAccept();
    });

    assert.deepEqual(store.getState().videos.data.get(expectedVideo.key), expectedVideo);
    sinon.assert.calledWith(updateVideoStub, video.key, {
      title: newTitle,
      description: newDescription,
    });
  });
```

Reply:
Equivalent test was added here: https://github.com/mitodl/odl-video-service/pull/223/files#diff-03e13d7c58734657b9875248878d3c94R88